### PR TITLE
updated to postgres 9.6.3

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Also making osx use built in kerberos (since our library causes problems at times)

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
Yeah I'll do that

**Comment by mariusvniekerk** [comment]
@ocefpaf are there any decent guides in being able to run the windows tests myself without appveyor?

**Comment by ocefpaf** [comment]
> @ocefpaf are there any decent guides in being able to run the windows tests myself without appveyor?

Not really. I am planning to write some though. You can use the [community](https://www.visualstudio.com/vs/community/) version of visual studio, but to be honest I don't recall what needs to be installed and there are way too many options there. They are not very intuitive BTW.

Then comes the annoying part, which is opening a terminal with `vcvarsall.bat x64`. I use portable apps terminal que set the `vcvarsall.bat x64` path to be called when I open it. You can navigate manually and run it if you don't want to use an external terminal software.

**Comment by gillins** [comment]
Just make sure you install the C++ part with the community edition of VS 2015 (for Python 3.5-3.6). I agree the install isn't straightforward, and takes a long time. You will need [this edition of VS 2008](https://www.microsoft.com/en-au/download/details.aspx?id=44266) for Python 2.7.

`conda build` will find and source `vcvarsall.bat` so if you are using that to do your builds you will find it a bit easier... Good luck.

**Comment by ocefpaf** [comment]
> `conda build will find and source vcvarsall.bat`

That did not happen in my setup :confused:  I wonder what I did wrong there...

**Comment by gillins** [comment]
> That did not happen in my setup

Hmmm. Do you get an error message about it from `conda build`? Did you install VS into the default location?

**Comment by ocefpaf** [comment]
> Did you install VS into the default location?

Not sure. I go to `C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build` to get it.

> Do you get an error message about it from conda build? 

I don't have the Windows machine here to check the message exactly. I will try again soon and compile some notes. Thanks!

**Comment by gillins** [comment]
It looks like you have installed VS 2017. `conda build` will be looking for VS 2015 for Python 3.5-3.6.

**Comment by ocefpaf** [comment]
> It looks like you have installed VS 2017. `conda build` will be looking for VS 2015 for Python 3.5-3.6.

Good to know. I will add that to my notes. Thanks again!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -58,3 +58,4 @@ extra:
     - gillins
     - msarahan
     - ocefpaf
+    - mariusvniekerk
```
Comment (ocefpaf) [inline recipe/meta.yaml]: :+1:

**Comment by ocefpaf** [comment]
> Also making osx use built in kerberos (since our library causes problems at times)

@mariusvniekerk can you elaborate on this so we can look into it?

**Comment by mariusvniekerk** [comment]
I have a kerkeros enabled postgresql instance that simply doesn't work on osx with the krb5 that we build.  Behavior is unfortunately a fairly opaque GSSAPI failure.  It does result in some strange kinit behaviors. 

Let me see if I can provide better logs

File: recipe/meta.yaml [hunk 2]
```diff
@@ -11,12 +11,12 @@ source:
   sha256: {{ sha256 }}
   patches:
     - fix_libraries_win.patch  # [win]
-    - timezone.patch  # [win and (py >= 35)]
+    # - timezone.patch  # [win and (py >= 35)]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Does this patch needs to be refreshed or was it solved upstream? If the former we need a new PR, if the latter let us just remove the commented out patch.

Comment (mariusvniekerk) [inline recipe/meta.yaml]: Seems like it was solved upstream.  I do vaguely recall reading that in the release notes.

**Pull request merged**